ExampleMod/Content/Buffs/ExampleMinecartBuff.cs

```diff
@@ -0,0 +1,31 @@
+using Terraria.ModLoader;
+using ExampleMod.Content.Mounts;
+using Terraria.Localization;
+using Terraria;
+using ExampleMod.Content.Items.Mounts;
+
+namespace ExampleMod.Content.Buffs
+{
+	//TODO 1.4.5: review MinecartLeft/Right if it still exists
+	public class ExampleMinecartBuff : ModBuff
+	{
+		// Use the vanilla DisplayName ("Minecart")
+		//public override LocalizedText DisplayName => Language.GetText("BuffName.MinecartLeft");
+		// But for the sake of example, we want to reuse the item name
+		public override LocalizedText DisplayName => ModContent.GetInstance<ExampleMinecart>().DisplayName;
+
+		// Use the vanilla Description
+		public override LocalizedText Description => Language.GetText("BuffDescription.MinecartLeft");
+
+		public override void SetStaticDefaults() {
+			Main.buffNoTimeDisplay[Type] = true;
+			Main.buffNoSave[Type] = true;
```

Comment:
In vanilla, these are automatically set via `BasicMountData` in `Main.Initialize_TileAndNPCData1`. Should we add a tML method to `BuffLoader` which copies these values across in the same manner in-between `SetupContent` and `PostSetupContent`?

To be clear, `BasicMountData` is what's being replaced with `BuffID.Sets.MountType` in 1.4.5 which will have the same behavior (just minus the direction flag).